### DISCUSSION TITLE
Previsão para versão 2.0?

### DISCUSSION THREAD

**Discussion by GuilhermeFCO** [diego3g/rsxp-2023#90] [General]
Olá pessoal, espero que estejam todos bem.

Queria muito participar de um projeto open source, ainda mais com a galera que acompanha a Rocketseat.
Temos alguma previsão para começar a versão 2.0?
Houve poucas tarefas para a quantidade de pessoas que querem participar.

Desde já agradeço a todos da Rocketseat e os seguidores.

**Comment by rpc1910** [comment]
Acredito que seja iniciada assim que terminar a 1.0
